ext/vscode/src/commands/devCenterMode.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { IActionContext } from '@microsoft/vscode-azext-utils';
+import * as vscode from 'vscode';
+import { createAzureDevCli } from '../utils/azureDevCli';
+import { execAsync } from '../utils/process';
+
+export function enableDevCenterMode(context: IActionContext): Promise<void> {
+    return setDevCenterMode(context, true);
+}
+
+export async function disableDevCenterMode(context: IActionContext): Promise<void> {
+    return setDevCenterMode(context, false);
+}
+
+async function setDevCenterMode(context: IActionContext, enable: boolean): Promise<void> {

```

Comment:
Just curious, @savannahostrowski, would we want to send a telemetry event for turning dev center on and off?

Reply:
Fortunately telemetry comes for free with commands using the `@microsoft/vscode-azext-utils` package. So already have telemetry on this. 👍